engine/table/src/main/java/io/deephaven/engine/table/impl/AbstractColumnSource.java

```diff
@@ -379,13 +378,9 @@ public void pushdownFilter(
     }
 
     @Override
-    public Map<String, String> renameMap(final WhereFilter filter, final ColumnSource<?>[] filterSources) {
-        // Default to returning an empty map
-        return Map.of();
-    }
-
-    @Override
-    public PushdownFilterContext makePushdownFilterContext() {
+    public PushdownFilterContext makePushdownFilterContext(
+            final WhereFilter filter,
+            final List<ColumnSource<?>> filterSources) {
```

Comment:
Much of this overlaps with #6893 but I needed functional code to run tests and they share a lot of corrections and refactoring.